Simplified/NYPLSettings.m

```diff
@@ -109,12 +98,13 @@ - (BOOL) userHasSeenFirstTimeSyncMessage
   return [[NSUserDefaults standardUserDefaults] boolForKey:userSeenFirstTimeSyncMessageKey];
 }
 
+// FIXME: This should be in `AccountsManager`, not `NYPLSettings`.
```

Comment:
This will have to wait for another day. The main issue is that so many parts of the app use `Int` account IDs instead of `Account` objects.